### ISSUE TITLE
Parallel jobs using dorado duplex?

### ISSUE THREAD

**Issue by SalvadorGJ** [nanoporetech/dorado#264]
Hi,

I have a directory with the `.pod5` files generated by a single flowcell with multiple samples sequenced together. As it is a lot of data and I have time constraints to enter into the HPC queue, I was wondering if it's possible to run `dorado duplex` with parallel jobs, by using subsamples with some files each. The issue that I could imagine is that the duplex pair may not be in the same file, but I'm not sure if that is true. Maybe other solution could be using `duplex_tools pair` as in previous releases, but also I don't know if that workflow is parallelizable. Could you please suggest me what should I do?  

Kind regards,
Salvador

**Comment by tijyojwad** [comment]
Hi @SalvadorGJ - one way you can split the files is by grouping the signals by channel. you can use the pod5 subset tool to do that. Instructions are here - https://pod5-file-format.readthedocs.io/en/latest/docs/install.html and https://pod5-file-format.readthedocs.io/en/latest/docs/tools.html#pod5-subset

e.g.
```
pod5 view pod5_dir/ -r -i "read_id,channel" > channels.txt
pod5 subset pod5_dir/ -r --table ./channels.txt --columns channel --output channel_subset
```

the output will be a folder called `channel_subset` which should have pod5s split by channel. you can run Dorado duplex in parallel on each of those subset pod5s.

if these files are still too large, you can also partition them by channel + mux.

**Comment by SalvadorGJ** [comment]
Hi @tijyojwad 

Sorry for the late reply. I got a bunch of issues trying the method you suggested, but finally I was able to find an efficient way to process my reads. I only want to indicate for whoever wants to make parallel jobs using `duplex`, that `pod5 subset` was extremely slow, so decided to continue with `pod5 filter`, which is in theory the same process: [pod5 subset killed](https://github.com/nanoporetech/pod5-file-format/issues/53). The only problem is that both `subset` and `filter` are not able to handle 20,000 files at once, so I needed to batch the input raw pod5 files, which considerably increases the number of jobs, and therefore the complexity of the process. 

Thanks again for your support!
